src/services/api-types.ts

```diff
@@ -99,13 +99,13 @@ export const InternationalTimeDistributionEntrySchema = zod.object({
 });
 
 export const TimeZipCodeDistributionEntrySchema = zod.object({
-  x: zod
-    .object({
-      week: YearWeekWithDaySchema,
-      zip_code: zod.string(),
-    })
-    .transform(v => ({ week: v.week, zipCode: v.zip_code })),
-  y: CountAndProportionWithCISchema,
+  x: zod.object({
+    week: YearWeekWithDaySchema,
+    zipCode: zod.string(),
```

Comment:
@chaoran-chen It looks like the documentation is wrong [here](https://github.com/cevo-public/cov-spectrum-docs/blob/develop/API.md#variant-time-zip-code-distribution).

Reply:
Thanks for pointing it out! Is fixed.